src/controllers/personsNameController.ts

```diff
@@ -0,0 +1,57 @@
+import { NextFunction, Request, Response } from "express";
+import * as config from "../config";
+import { validationResult } from "express-validator";
+import { formatValidationError, getPageProperties } from "../validations/validation";
+import { BASE_URL, PERSONS_NAME, PERSONAL_CODE } from "../types/pageURL";
+import { Session } from "@companieshouse/node-session-handler";
+import { USER_DATA } from "../utils/constants";
+import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../utils/localise";
+import { saveDataInSession } from "../utils/sessionHelper";
+import { ClientData } from "../model/ClientData";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    const clientData: ClientData = session.getExtraData(USER_DATA) ? session.getExtraData(USER_DATA)! : {};
+    const payload = {
+        "first-name": clientData.firstName,
```

Comment:
Why are we using this format in payload first-name, last-name etc? why not use camel case?